### DISCUSSION TITLE
Is cider updating AM stats ?

### DISCUSSION THREAD

**Discussion by qLVN** [ciderapp/Cider#1613] [Q&A] [answered]
Hello, 

I have the Song Infos app on my phone to see my AM stats but I'm pretty sure that the app only has stats from my phones AM app or from music.apple.com

I don't know if it's normal or if I have to enable something so that Cider send it's stats to AM.

if you know anything about that, i would love to know more

**Comment by tva2000hn** [comment, marked as answer]
Yes, i got Replay from mostly using Cider.
